Source/Blazorise/Components/Dropdown/Dropdown.razor.cs

```diff
@@ -77,7 +78,7 @@ protected override void OnAfterRender( bool firstRender )
         {
             if ( firstRender )
             {
-                JSModule.Initialize( ElementRef, ElementId, new
+                JSModule.Initialize( ElementRef, ElementId, buttonList?.FirstOrDefault()?.ElementId, new
```

Comment:
Is it always the first occurrence?

Reply:
What do you mean by the first occurrence? The idea is to initialize it so that is already ready by the time we click on a toggle button. This way it will be faster to show and will not do the weird `pop-out`.